lib/mayaUsd/fileio/utils/meshWriteUtils.h

```diff
@@ -29,11 +29,75 @@
 #include <pxr/usd/usdGeom/mesh.h>
 #include <pxr/usd/usdUtils/sparseValueWriter.h>
 
+#include <maya/MBoundingBox.h>
 #include <maya/MDagPath.h>
 #include <maya/MFnMesh.h>
 #include <maya/MObject.h>
 #include <maya/MString.h>
 
+/**
+ * Gets the minimum unique name of a DAG node.
+ *
+ * @param node  The node to find the name of.
+ *
+ * @return      The name as a Maya string.
+ */
+MString mayaGetUniqueNameOfDAGNode(const MObject& node);
```

Comment:
Noted this in the .cpp, but is the minimal uniqueness important, or can we remove this and just use `UsdMayaUtil::GetMayaNodeName()`?

Reply:
As above: I think it potentially could be if you had multiple target meshes with the same names under different transform groups. In that case, we could have them saved with the same name in the resulting usd file, but it might make more sense to have them saved with the minimum unique name so that it's clear which blendshape target is which.